src/annotator/config/config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var settings = require('./settings');
+var sharedSettings = require('../../shared/settings');
+
+/**
+ * Reads the Hypothesis configuration from the environment.
+ *
+ * @param {Window} window_ - The Window object to read config from.
+ */
+function configFrom(window_) {
+  var config = {};
+
+  config.app = settings.iFrameSrc(window_.document);
```

Comment:
This is how I want all the config settings in this file to work in the future: the `config.app` setting is mentioned by name in `config.js`, and it calls a separate function that returns the setting's value.